sudoku/general/onInputCheck.js

```diff
@@ -0,0 +1,20 @@
+let wrapper = document.querySelector('.board');
+let check = document.querySelector('.check');
+let hard = document.querySelector('.hard');
+let checkHard = document.querySelector('.addRandom');
+function checkNum(event) {
+  const onlyNumbers = /^[1-9]$/;
+
+  if (!event.target.innerHTML.match(onlyNumbers) && event.target.innerHTML != '') {
```

Comment:
Empty string is a falsy value so you can write like this:
```suggestion
  if (!event.target.innerHTML.match(onlyNumbers) && event.target.innerHTML) {
```